vtm.js

```diff
@@ -193,22 +193,35 @@ const app = Vue.createApp({
     };
   },
   methods: {
-    /** will update data based on event received from child component */
+    /** 
+     * will update data based on event received from child component 
+     * input parameter statChangeData is array identifying which stat needs to be changed to which value
+     * for example
+     * [ "Attributes", "Physical", "Strength", 5]
+    */
     updateStat(statChangeData) {
-      debugger;
-      for (var statSection = 0; statSection < data.length; statSection++) {
-        if (data[statSection].id == statChangeData[0]) {
-          for (var statCategory = 0; statCategory < data[statSection].list.length; statCategory++) {
-            if (data[statSection].list[statCategory].id == statChangeData[1]) {
-              for (var stat = 0; stat < data[statSection].list[statCategory].list.length; stat++) {
-                if (data[statSection].list[statCategory].list[stat].id == statChangeData[2]) {
-                  data[statSection].list[statCategory].list[stat].value = statChangeData[3];
-                }
+      if (statChangeData[0] == "Attributes") {
```

Comment:
@pmlich @longnicknam3 toto neni vobec pekne, ale neviem ako to obist

Reply:
no to je celkom silena funkcia... mozno by sa dal pouzit switch ale tez by to nevypadalo pekne

Reply:
toto by sa mozno dalo upravit, ak by som emity robil inac. Alebo mozno keby som si definoval nejake dalsie funkcie, tak by to vypadalo lepsie